src/src/ESPEasyCore/ESPEasyWifi.cpp

```diff
@@ -560,6 +560,21 @@ void resetWiFi() {
   initWiFi();
 }
 
+#ifdef ESP32
+void removeWiFiEventHandler()
+{
+  WiFi.removeEvent(wm_event_id);
+}
+
+void registerWiFiEventHandler()
+{
+#if defined(ESP32)
```

Comment:
Checked twice? (looking at line 563)

Reply:
Code came from somewhere else and at 1am I was probably not as sharp anymore ;)